src/test/java/pages/NewAccountModalPage.java

```diff
@@ -38,6 +44,23 @@ public void fillInAccountForm(NewAccountModel newAccount) {
         new TextInput(driver, "Shipping State/Province").inputText(newAccount.getBillingState());
         new TextInput(driver, "Shipping Zip/Postal Code").inputText(newAccount.getBillingZip());
         new TextInput(driver, "Shipping Country").inputText(newAccount.getBillingCountry());
+        LOGGER.info(String.format("Entered data: " + "\nname: " + newAccount.getAccountName()
+                                    + "\nphone: " + newAccount.getPhone()
+                                    + "\nType: " + newAccount.getType()
+                                    + "\nDescription: " + newAccount.getDescription()
+                                    + "\nWebsite: " + newAccount.getWebsite()
+                                    + "\nIndustry: " + newAccount.getIndustry()
+                                    + "\nEmployees: " + newAccount.getEmployees()
+                                    + "\nBilling Street: " + newAccount.getBillingStreet()
+                                    + "\nBilling City: " + newAccount.getBillingCity()
+                                    + "\nBilling State/Province: " + newAccount.getBillingState()
+                                    + "\nBilling Zip/Postal Code: " + newAccount.getBillingZip()
+                                    + "\nBilling Country: " + newAccount.getBillingCountry()
+                                    + "\nShipping Street: " + newAccount.getBillingStreet()
+                                    + "\nShipping City: " + newAccount.getBillingCity()
+                                    + "\nShipping State/Province: " + newAccount.getBillingState()
+                                    + "\nShipping Zip/Postal Code: " + newAccount.getBillingZip()
+                                    + "\nShipping Country: " + newAccount.getBillingCountry()));
```

Comment:
Please add toString method to model you can do it manually or with help of lombok, and then call it like this LOGGER.info(String.format("Entered data: " + newAccount.toString();

Reply:
Done